### PR TITLE
Changed provider of SmartOS from Joyent to MNX

### PR DESCRIPTION
**Pull request by brianewell**
Updated the distribution list to reflect the changed ownership of SmartOS from Joyent to MNX Solutions.

### CODE REVIEW

File: docs/about/distro.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ For major feature differences, the following chart may be helpful.
...
 | [OpenIndiana](https://www.openindiana.org/)    | [OpenIndiana](https://www.openindiana.org/)                      | Workstation, Servers | Ubuntu           | <i class="material-icons">check</i> | IPS                                 | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> |                                     |
-| [SmartOS](https://smartos.org)                 | [Joyent](https://www.joyent.com)                                 | Hypervisor           | CoreOS           | <i class="material-icons">check</i> | [pkgsrc](https://pkgsrc.joyent.com) | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> | <i class="material-icons">check</i> |
+| [SmartOS](https://smartos.org)                 | [MNX Solutions](https://mnx.io/)                                 | Hypervisor           | CoreOS           | <i class="material-icons">check</i> | [pkgsrc](https://pkgsrc.joyent.com) | <i class="material-icons">check</i> |                                     | <i class="material-icons">check</i> | <i class="material-icons">check</i> |
```
Comment (danmcd) [inline docs/about/distro.md:25]: Make the URL point to https://mnxsolutions.com instead  please.  :)

Comment (brianewell) [reply docs/about/distro.md]: Done

**Review by danmcd — APPROVED** [review]

**Comment by danmcd** [comment]
At this point someone with write access (e.g. @jclulow or @bdha perhaps) can push.

**Comment by richlowe** [comment]
Integrated, thanks!

**Pull request closed without merging**